x/bandwidth/internal/types/params.go

```diff
@@ -8,15 +8,28 @@ import (
 
 // Default parameter values
 const (
-	DefaultLinkMsgCost                                int64   = 100
-	DefaultRecoveryPeriod                             int64   = 18000
-	DefaultAdjustPricePeriod                          int64   = 10
+	// DefaultParamspace default name for parameter store
+	DefaultParamspace = ModuleName
+
+	DefaultLinkMsgCost                                int64  = 100
+	DefaultRecoveryPeriod                             int64  = 18000
+	DefaultAdjustPricePeriod                          int64  = 10
 	DefaultBaseCreditPrice                            string = "1.0"
-	DefaultDesirableNetworkBandwidthForRecoveryPeriod int64   = 200000000
-	DefaultTxCost                                     int64   = 300
-	DefaultNonLinkMsgCost                             int64   = 500
-	DefaultSlidingWindowSize                          int64   = 18000
+	DefaultDesirableNetworkBandwidthForRecoveryPeriod int64  = 200000000
+	DefaultTxCost                                     int64  = 300
+	DefaultNonLinkMsgCost                             int64  = 500
+	DefaultSlidingWindowSize                          int64  = 18000
 	DefaultShouldBeSpentPerSlidingWindow              string = "200000000"
+
+	MinLinkMsgCost       = 1
+	MinRecoveryPeriod    = 100
+	MinAdjustPricePeriod = 1
+	//MinBaseCreditPrice
+	//MinDesirableNetworkBandwidthForRecoveryPeriod
+	MinTxCost                        = 1
+	MinNonLinkMsgCost                = 1
+	MinSlidingWindowSize             = 100
+	MinShouldBeSpentPerSlidingWindow = 1000
```

Comment:
SlidingWindowSize and ShouldBeSpentPerSlidingWindow params should be initialized from RecoveryPeriod and DesirableNetworkBandwidthForRecoveryPeriod params, not from genesis.